cclib/method/volume.py

```diff
@@ -149,29 +155,36 @@ def getbfs(coords, gbasis):
         }
 
     bfs = []
-    for i, atom in enumerate(mymol):
-        bs = gbasis[i]
-        for sym, prims in bs:
-            for power in sym2powerlist[sym]:
-                bf = CGBF(atom.pos(), power)
-                for expnt, coef in prims:
-                    bf.add_primitive(expnt, coef)
-                bf.normalize()
-                bfs.append(bf)
+    
+    #PyQuante
+    if _found_pyquante:

```

Comment:
It would be better to have this guard the full definition of `getbfs`, rather than call the function and have nothing happen. (I know that's not how it's done in other parts of the code.)

Reply:
If `_found_pyquante` or `_found_pyquante2` (in the next PR) are not `True`, `cclib2pyquante.makepyquante`, which is the first line to be executed in both functions should trigger an Exception. Would the exception be enough? Adding something like `if not _found_pyquante: raise someException` could be more explicit?

Reply:
No, the exception is fine for now.